src/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
rename this file to routes.tsx and export it * in the index.ts file that you have to create in the routes folder